code/modules/guardian/abilities/minor/teleport.dm

```diff
@@ -0,0 +1,99 @@
+/datum/guardian_ability/minor/teleport
+	name = "Teleportation Pad"
+	desc = "The guardian can prepare a teleportation pad, and teleport things to it afterwards."
+	cost = 3
```

Comment:
should probably lower this

Reply:
no because giving everyone with a stand mapwide teleportation for 1-2 points is a pog moment